foundation_cms/legacy_apps/mozfest/templates/partials/primary_hero.html

```diff
@@ -18,6 +18,14 @@ <h1 class="tw-h1-heading">{% firstof page.banner_heading page.title %}</h1>
                 {% if page.banner_link_url and page.banner_link_text %}
                     <a class="tw-btn-secondary" href="{{ page.banner_link_url }}" data-cta-event="hero">{{ page.banner_link_text }}</a>
                 {% endif %}
+
+                {% if page.hero_cta %}
+                    <!--  How we usually render blocks-->
+                    {{ page.hero_cta.0 }}
+                    <!-- Another possible option. To workaround request problem.  Though Streamfield block base template forces a "justify center" class -->
+                    {% include "./tito_widget_block.html" with value=page.hero_cta.0.value request=request %}
```

Comment:
This again was an idea for a workaround with the request issue. Also, I wanted to note that the `tito_widget_block` template extends from `base_streamfield`, which justifies the content in center.

Once I got the button to render as expected, it was in the middle of the div. Maybe we can wrap it in a container div and make the size max-content and left-aligning?